File: backend/Restaurant.API/Restaurant.Models/Menu.cs
using System;
using System.Collections.Generic;

namespace Restaurant.Models;

public partial class Menu
{
    public int dish_id { get; set; }

    public int cuisine_rest { get; set; }

    public string dish { get; set; } = null!;

    public decimal price { get; set; }

    public virtual Restaurants CuisineRestNavigation { get; set; } = null!;
}

File: frontend/Restaurant.UI/src/app/models/menu.ts
export class Menu{
    dish_id: number;
    rest_scoreid: number;
    dish: string;
    price: number;


    constructor (dish_id: number, rest_scoreid: number, dish: string, price: number){
        this.dish_id = dish_id;
        this.rest_scoreid = rest_scoreid;
        this.dish = dish;
        this.price = price;
    }
}